x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('EngineOverview', () => {
     it('isLoading', () => {
       // We use render() instead of mount() here to not trigger lifecycle methods (i.e., useEffect)
       // TODO: Consider pulling this out to a renderWithContext mock/helper
-      const wrapper = render(
+      const wrapper: Cheerio = render(
```

Comment:
Why don't we have to import `Cheerio`?

Reply:
Short answer: I have no idea lmao :dead_inside:

Long answer: I think it's a global declared by either Jest or Enzyme. This threw me for a massive loop as well and I spent a good amount of time trying to figure out WTF wrapper type it wanted. I tried ReactWrapper & ShallowWrapper (threw type errors). Enzyme's docs lists `render()` returning a [CheerioWrapper](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/render.html) (even though the URL structure is nested under ShallowWrapper??) but every time I tried to import CheerioWrapper from Enzyme I got a "export does not exist error".

I eventually tried YOLOing this and it worked and had also spent enough time on it that just moved on 🤦‍♀️ Anyway, this is why people avoid `render()` haha.